### DISCUSSION TITLE
新增生僻字模型

### DISCUSSION THREAD

**Discussion by shiyutang** [PaddlePaddle/PaddleOCR#12738] [Ideas & Features]
## 背景

经过需求征集https://github.com/PaddlePaddle/PaddleOCR/issues/10334 和每周技术研讨会 https://github.com/PaddlePaddle/PaddleOCR/issues/10223 讨论，我们确定了新增生僻字模型的任务。

## 解决步骤

1. 替换现有字典txt为扩充《通用规范汉字表》的字典。
2. 在现有数据集上通过数据合成copy paste等方式实现语料的平衡，并重新训练PPOCRV3的检测和识别模型。
3. 对比训练后模型在普通文字和生僻字上的检测、识别精度，并和PPOCRV3模型最优模型进行对比；达到普通字精度不变或者更高，生僻字上精度进一步提升的效果。
5. 提交PR到ppocr，替换最优模型。

**Comment by hademen** [comment]
生僻字模型的模型在哪里可以下载

**Comment by shiyutang** [comment]
这是一个命题任务，可以报名参与，使用aistudio资源进行训练。训练之后就在相关PR中公布。

**Comment by zhoutianrui-tongji** [comment]
请问现在有生僻字模型了吗？

**Comment by zhuxiaobin** [comment]
@shiyutang 生僻字模型进度咋样了？

**Comment by dc6273632** [comment]
生僻字模型有消息了吗？

**Comment by pangmaoran** [comment]
生僻字模型进度如何了？

**Comment by Mike575** [comment]
生僻字模型进度如何了？

**Comment by adamzhg** [comment]
生僻字模型进度如何了？
